driver/driver.go

```diff
@@ -14,16 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-//go:generate mockgen -package=driver -destination=driver.mock.go github.com/container-storage-interface/spec/lib/go/csi IdentityServer,ControllerServer,NodeServer
```

Comment:
I think this is an old version of this file. I remember fixing a lot of this.

Reply:
@lpabon not sure what to say, it was in csi-test master.

Reply:
Looks like you may want to rebase your branch. Check out master:
https://github.com/kubernetes-csi/csi-test/blob/f56f43f4866e6bdef38e74e69c047b8377e0221c/driver/driver.go#L17